README.md

```diff
@@ -214,18 +214,11 @@ The array passed to `FormHelper::create()` will cause the helper to create an
 query params.
 
 #### Adding a reset button dynamically
-If you pass down the information on whether the query was modified by your
-search query strings, you can also include a reset button only if necessary:
-
-```php
-// in your controller action
-$this->set('isSearch', $this->Articles->isSearch());
-```
-
-Then one can switch based on that in the template:
+Since the Prg component will pass down the information on whether the query was modified by your
```

Comment:
It would be better if this statement was reworded to explicitly mention that `$_isSearch` view var is auto set if a search has been performed. This would make is more clear for newbies.